evap/evaluation/templates/navbar.html

```diff
@@ -36,19 +36,29 @@
                     <li class="nav-item"><a class="nav-link" href="{% url 'results:index' %}">{% trans 'Results' %}</a></li>
                 {% endif %}
                 {% if user.is_manager %}
```

Comment:
i like the new menu! it would be even better with some additional rearrangements. could you please order it as follows (renaming in **bold**):
1. Overview
2. Semesters ...
3. Results
4. Users ...
    1. **User list**
    2. Create
    3. Import
    4. Merge
    5. Update 
5. Questionnaires
6. Publish grades ...
7. More ...
    1. Course Types
    2. Degrees
    3.** Email templates**
    4. FAQ
    5. Reward point redemption events